cloud_governance/policy/azure/cost_billing_reports.py

```diff
@@ -34,21 +39,38 @@ def get_common_data(self):
         upload_data['Account'] = self.azure_operations.account_name+'-'+self.azure_operations.cloud_name.title()
         upload_data['Budget'] = 0
         upload_data['AllocatedBudget'] = 0
-        upload_data['CostCenter'] = self.__cost_center
+        upload_data['CostCenter'] = int(self.__cost_center)
         upload_data['CloudName'] = self.azure_operations.cloud_name
+        upload_data['Owner'] = 'Others'
         return upload_data
 
     @logger_time_stamp
-    def get_data_from_costs(self, cost_data_rows: list, cost_data_columns: list, cost_type: str, cost_billing_data: dict):
+    def get_data_from_costs(self, cost_data_rows: list, cost_data_columns: list, cost_type: str, cost_billing_data: dict, subscription_id: str = '', account_name: str = '', cost_center: int = 0):
         """
         This method get the data from the cost
         @param cost_billing_data:
         @param cost_type:
         @param cost_data_rows:
         @param cost_data_columns:
+        @param subscription_id:
         @return:
         """
-        common_data = self.get_common_data()
+        common_data = self.__common_data
+        index_id = ''
+        common_data["AccountId"] = subscription_id
+        if account_name:
+            common_data['Account'] = account_name
+        if cost_center > 0:
+            common_data['CostCenter'] = cost_center
+        if subscription_id:
+            cost_center, allocated_budget, years = self.update_to_gsheet.get_cost_center_budget_details(account_id=subscription_id, dir_path='/tmp')
+            if cost_center:
+                common_data['CostCenter'] = int(cost_center)
+                common_data['Owner'] = 'Shai'
```

Comment:
Can you make 'Shai' environment variable / constant ?

Reply:
Added const varaibles